src/go/rpk/pkg/cli/cmd/acl/delete.go

```diff
@@ -69,7 +68,7 @@ resource names:
 
 			var printDeletionsHeader bool
 			if !noConfirm || dry {
-				describeReqResp(adm, printAllFilters, true, b)
+				describeReqResp(adm, printAllFilters, true, b, "text")
```

Comment:
(making this comment once, but this applies globally):
Can this constant be moved into out, `const FmtText = "text"` and all of these `"text"` usages be replaced with `out.FmtText`? Mostly to ensure we don't have a typo at some point.

Reply:
For consistency, there's a few instances of "text" being the default for --format that are outside this PR, such as pkg/cli/cmd/cluster/license/info.go that I'm going to update with the new out.FmtText variable. Hopefully this is okay. If not I'll undo that.

Reply:
Related to --format flags outside this PR like the one above, there's structs in here that only have json tags, and json martial printing locally in the file. This these places predate this PR so that's not a surprise, but does the team want me to refactor these existing spots to use the same centralized out.PrintFormatted() function I've added, as well as add yaml tags?